### PR TITLE
Adding permission for resend-code endpoint at toml level

### PR DESCRIPTION
**Pull request by indeewari**
## Purpose
Provide instructions for upgrading the permission level of the resend-code endpoint in IS 5.11.0.

## Goals
Upgrade the permission level following the documented instructions to ensure the stability of the existing system.

### CODE REVIEW

File: en/identity-server/5.11.0/docs/develop/using-the-self-sign-up-rest-apis.md [hunk 1]
```diff
@@ -6,3 +6,18 @@
 
 !!! info "Related Links" 
     For information on self-registration via the UI instead, see [Self-Registration and Account Confirmation](../../learn/self-registration-and-account-confirmation).
+
+## Enhance the default permission for the resend-code endpoint.
+
+Update the default permission for the resend-code endpoint in the Self-Sign-up API to utilize identity management permissions prior to production deployment.
```
Comment (himeshsiriwardana) [inline en/identity-server/5.11.0/docs/develop/using-the-self-sign-up-rest-apis.md:12]:
```suggestion
The [resend-code endpoint](https://api-docs.wso2.com/apidocs/is/is511/selfregister-v5.11.0/#!/operations#SelfRegister#meResendCodePost) of the self sign-up rest APIs is used to resend the confirmation code to an authenticated user. While no scopes are required to invoke this API by default, we recommend limiting permissions to this endpoint using scopes prior to production deployment.

To do so, add the following configurations to the `deployment.toml` file
```

File: en/identity-server/5.11.0/docs/develop/using-the-self-sign-up-rest-apis.md [hunk 2]
```diff
@@ -6,3 +6,18 @@
...
+
+Add the following properties to the `deployment.toml` file.
```
Comment (himeshsiriwardana) [inline en/identity-server/5.11.0/docs/develop/using-the-self-sign-up-rest-apis.md:14]:
```suggestion
```

File: en/identity-server/5.11.0/docs/develop/using-the-self-sign-up-rest-apis.md (hunk 1 above)
Comment (indeewari) [reply en/identity-server/5.11.0/docs/develop/using-the-self-sign-up-rest-apis.md:16]: Updated with 4c04088b7637768f658b714e2b8494dcdf6e4ae6

File: en/identity-server/5.11.0/docs/develop/using-the-self-sign-up-rest-apis.md (hunk 2 above)
Comment (indeewari) [reply en/identity-server/5.11.0/docs/develop/using-the-self-sign-up-rest-apis.md:18]: Updated with 4c04088b7637768f658b714e2b8494dcdf6e4ae6

File: en/identity-server/5.11.0/docs/develop/using-the-self-sign-up-rest-apis.md [hunk 3]
```diff
@@ -6,3 +6,22 @@
...
 !!! info "Related Links" 
     For information on self-registration via the UI instead, see [Self-Registration and Account Confirmation](../../learn/self-registration-and-account-confirmation).
+
+## Enhance the default permission for the resend-code endpoint.
+
+The [resend-code endpoint](https://api-docs.wso2.com/apidocs/is/is511/selfregister-v5.11.0/#!/operations#SelfRegister#meResendCodePost) of the self sign-up rest APIs is used to resend the confirmation code to an authenticated user. While no scopes are required to invoke this API by default, we recommend limiting permissions to this endpoint using scopes, prior to production deployment.
+
+To do so, add the following configurations to the `deployment.toml` file
+
+Update the default permission for the resend-code endpoint in the Self-Sign-up API to utilize identity management permissions prior to production deployment.
```
Comment (himeshsiriwardana) [inline en/identity-server/5.11.0/docs/develop/using-the-self-sign-up-rest-apis.md:16]:
```suggestion
```

File: en/identity-server/5.11.0/docs/develop/using-the-self-sign-up-rest-apis.md [hunk 4]
```diff
@@ -6,3 +6,22 @@
...
+
```
Comment (himeshsiriwardana) [inline en/identity-server/5.11.0/docs/develop/using-the-self-sign-up-rest-apis.md:17]:
```suggestion
```

File: en/identity-server/5.11.0/docs/develop/using-the-self-sign-up-rest-apis.md [hunk 5]
```diff
@@ -6,3 +6,22 @@
...
+Add the following properties to the `deployment.toml` file.
```
Comment (himeshsiriwardana) [inline en/identity-server/5.11.0/docs/develop/using-the-self-sign-up-rest-apis.md:18]:
```suggestion
```

**Review by himeshsiriwardana — APPROVED** [review]

**Pull request merged**